content/product/discover/users.md

```diff
@@ -66,12 +66,12 @@ Your partners likely have an incomplete view of the current state or what their
 
 - [How to conduct user research [1]](https://methods.18f.gov/): 18F Methods provide instruction on several different research activities to consider, including [interviews](https://methods.18f.gov/stakeholder-and-user-interviews), [contextual inquiries](https://methods.18f.gov/contextual-inquiry), [heuristic evaluation](https://methods.18f.gov/heuristic-evaluation), and more.
 
-- [How to conduct user research [2]](https://handbook.tts.gsa.gov/research-guidelines/): Guidelines on how research is done at TTS, including informing and getting consent, and managing Personally Identifiable Information (PII).
+- [How to conduct user research [2]](https://guides.18f.gov/ux-guide/research/): Guidelines on how research is done at TTS, including informing and getting consent, and managing Personally Identifiable Information (PII).
 
 - [How to synthesize user research](https://methods.18f.gov/): 18F Methods provide instructions on several different ways to tell users’ stories, including [journey maps](https://methods.18f.gov/journey-mapping), [user scenarios](https://methods.18f.gov/user-scenarios), [storyboards](https://methods.18f.gov/storyboarding), and [task flow analysis](https://methods.18f.gov/decide/task-flow-analysis/).
 
 - Examples of user journeys: [Example 1](https://docs.google.com/drawings/d/1SqxnC-smn2nE7tCk3_79YpPH5XlcltKPN3nsLf8F6PU/edit), Example 2 (<a href="https://app.mural.co/t/gsa6/m/gsa6/1611637245217/2d6ef2c9a78cdc2849c07edaf26d855381f50975" class="private-link">detailed</a>, [summary](https://docs.google.com/presentation/d/1G2jx7adxkDifD_wihWD3Ak1xnxH0ePQVJWH6RL7ibQs/edit#slide=id.g4e82d6a821_2_101))
 
 - [How to share user research](https://18f.gsa.gov/2018/02/06/getting-partners-on-board-with-research-findings/): How to present findings to your partner and allow room for reaching consensus before making recommendations.
 
-- [What is human-centered design](https://docs.google.com/document/d/1njVM3rSV_1Tyd7emSK9gMnbv_vzi0Dhm9vhE7LiqLrs/edit): A one-pager introducing and defining human-centered design and explaining why it’s a good idea.
+- [What is human-centered design](https://guides.18f.gov/ux-guide/): A guide introducing and defining human-centered design and explaining why it’s a good idea.
```

Comment:
This page does not actually define or introduce the term human-centered design

Reply:
thx for catching this! changed the title and desc to match the link

Reply:
Good point! How about this: 

`[What is human-centered design](https://guides.18f.gov/methods/about/#the-basics-of-human-centered-design): A high-level introduction to to human-centered design with links to further resources.` 

?

Reply:
Sorry I had the tab open and added comments after Mel addressed things. May no longer be relevant!